question/classes/output/qbank_chooser.php

```diff
@@ -51,6 +53,8 @@ class qbank_chooser extends \core\output\chooser {
      * @param context $context The relevant context.
      */
     public function __construct($real, $fake, $course, $hiddenparams, $context) {
+        debugging('Class qbank_chooser has been deprecated and moved to bank/editquestion,
```

Comment:
Can you please comment this and add to a TODO in tracker: https://tracker.moodle.org/browse/MDL-71679

```
Class qbank_chooser has been deprecated and moved to bank/editquestion, please use qbank_editquestion\qbank_chooser instead.
line 57 of /question/classes/output/qbank_chooser.php: call to debugging()
line 106 of /question/classes/output/qbank_chooser.php: call to core_question\output\qbank_chooser->__construct()
line 1072 of /mod/quiz/classes/output/edit_renderer.php: call to core_question\output\qbank_chooser::get()
line 126 of /mod/quiz/classes/output/edit_renderer.php: call to mod_quiz\output\edit_renderer->question_chooser()
line 211 of /mod/quiz/edit.php: call to mod_quiz\output\edit_renderer->edit_page()
```